### PR TITLE
Fix getTotalCount() method

### PR DESCRIPTION
**Pull request by curquiza**
Remove `nbHits` usage because this is not reliable information.

The Meili team is aware of this. Here are the different issues and comments about it to explain why it's confusing, and why we should not use it:
- https://github.com/meilisearch/MeiliSearch/issues/1120
- https://github.com/meilisearch/documentation/issues/561
- https://github.com/meilisearch/meilisearch-php/issues/119#issuecomment-732908568

TLDR;
`nbHits` is not reliable for pagination because can be exhaustive or not, depending on the value of `exhaustiveNbHits` that MeiliSearch returns which is always `false` for the moment.

We are sorry for this. We all hope this confusion will be fixed asap in MeiliSearch.

⚠️ The linter error in the CI will be fixed with #82

### CODE REVIEW

**Review by shokme — APPROVED** [review]
Ok nice thanks

**Comment by curquiza** [comment]
bors merge
